src/Templating/Helper/BlockHelper.php

```diff
@@ -341,39 +341,10 @@ protected function computeAssets(BlockContextInterface $blockContext, array &$st
         ];
     }
 
-    private function startTracing(BlockInterface $block): array
-    {
-        if (null !== $this->stopwatch) {
-            $this->traces[$block->getId()] = $this->stopwatch->start(
-                sprintf('%s (id: %s, type: %s)', $block->getName(), $block->getId(), $block->getType())
-            );
-        }
-
-        return [
-            'name' => $block->getName(),
-            'type' => $block->getType(),
-            'duration' => false,
-            'memory_start' => memory_get_usage(true),
-            'memory_end' => false,
-            'memory_peak' => false,
-            'cache' => [
-                'keys' => [],
-                'contextual_keys' => [],
-                'handler' => false,
-                'from_cache' => false,
-                'ttl' => 0,
-                'created_at' => false,
-                'lifetime' => 0,
-                'age' => 0,
-            ],
-            'assets' => [
-                'js' => [],
-                'css' => [],
-            ],
-        ];
-    }
-
-    private function stopTracing(BlockInterface $block, array $stats): void
```

Comment:
All the methods that were marked with `@internal` at 3.x should remain private in 4.x.

Reply:
Oh ok sorry